kiezcoding.html

```diff
@@ -84,7 +84,7 @@
 
             <h2>How much is it?</h2>
 
-            <p>Regular co-working day passes at co.up are 15 euro per day. For our special event, there will be a need-based sliding scale from 0-15 euro. If you are able to, <a href="https://members.co-up.de/membership_signup/new?plan_id=261dfabc9ec1ebf95aefa14246b80c4c">register for a regular priced co.up day pass</a>. If you are not able to pay the full price for any reason, <a href="mailto:WHICH_EMAIL_ADDRESS">email us to make the arrangements</a>.</p>
+            <p>Regular co-working day passes at co.up are 15 euro per day. For our special event, there will be a need-based sliding scale from 0-15 euro. If you are able to, <a href="https://members.co-up.de/membership_signup/new?plan_id=261dfabc9ec1ebf95aefa14246b80c4c">register for a regular priced co.up day pass</a>. If you are not able to pay the full price for any reason, please <a href="mailto:info@berlinjs.org">email us to make the arrangements</a>.</p>
```

Comment:
I have strong opinions on the unnecessary use of “please” in copy so brace yourself ;)

Adding “please” sounds condescending to me. They are not doing *us* any favour, so there is no need for “please”. We are providing instructions on how to accomplish something. Punkt. By adding “please” we imply that they are doing us a courtesy, which in turn implies there is a way to do this that is not courteous, which makes the whole thing an anxiety-rich minefield of unanswerable questions about proper conduct.

*If* anything needs a “please” is the the line before that, were we ask them to register and pay using the site.

But I’m fine with leaving it ;). Maybe this opinion deserves a blog post.